src/ResourceManager/Profile/Commands.Profile/help/Set-AzureRmEnvironment.md

```diff
@@ -438,61 +485,31 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -TrafficManagerDnsSuffix
-Specifies the domain-name suffix for Azure Traffic Manager services.
-
-```yaml
-Type: String
-Parameter Sets: Name
-Aliases: 
-
-Required: False
-Position: 12
-Default value: None
-Accept pipeline input: True (ByPropertyName)
-Accept wildcard characters: False
-```
-
-### -BatchEndpointResourceId
-The resource identifier of the Azure Batch service that is the recipient of the requested token
+### -StorageOAuthEndpointResourceId
+The audience for tokens authenticating with Azure Storage.
 
 ```yaml
 Type: String
 Parameter Sets: (All)
 Aliases: 
 
 Required: False
-Position: 20
+Position: 23
```

Comment:
This is a non mandatory param... can you delete the position on it?

Reply:
I add position since following other parameters in the cmdlets.
fixed by removing it.